src/components/ProtectionBlockInputs.tsx

```diff
@@ -46,27 +45,6 @@ export default function ProtectionBlockInputs<
     })
   }
 
```

Comment:
@TahaKhanAbdalli - can you explain why we've deleted the following block of code please?

Reply:
This is `pMarking`, that we discussed with salony and it needed to be removed becuase our db has protectiveMarking not pMarkin.